### DISCUSSION TITLE
StorageRPCSecret API key - have a question

### DISCUSSION THREAD

**Discussion by stuberman** [filecoin-project/curio#27] [Q&A] [answered]
In the default Base layer there is a section for API's with a sub category of **StorageRPCSecret**.
The tool tip states: 
### [Apis]

>    RPC Secret for the storage subsystem.
>    If integrating with lotus-miner this must match the value from
>    cat ~/.lotusminer/keystore/MF2XI2BNNJ3XILLQOJUXMYLUMU | jq -r .PrivateKey
>   
>    type: string
>   StorageRPCSecret = ""

Curio acts like a lotus-miner, so if there is no remote node running lotus-miner, then this field can be disabled?
In today's version of Curio, would it not make sense to run another instance of Curio on a storage array?
I noticed that in the local Curio repo there is no keystore.

**Comment by LexLuthr** [comment, marked as answer]
We should probably remove that line at this point. Earlier, `lotus-miner` could become just another node for Curio.

Curio is the sole owner of everything now. All storage and hardware should be directly attached to Curio nodes. Also, this field cannot be disabled, it is the main "key" used to API and storage auth. 

There is no local `keystore` and the key now lives in DB (the field we are discussing).

**Comment by snadrus** [comment]
done.
